src/backend/core/authentication/backends.py

```diff
@@ -88,6 +90,11 @@ def get_or_create_user(self, access_token, id_token, payload):
         sub = user_info["sub"]
         email = user_info.get("email")
 
+        if settings.FEATURES_ALPHA:
+            alpha_enabled = posthog.feature_enabled("alpha", email)
+            if not alpha_enabled:
+                raise EmailNotWhitelisted("Email not authorized for alpha")
```

Comment:
```suggestion
                raise EmailNotWhitelisted()
```

Once the exception, I think the message can be also removed.